tests/UnitTests/Foundation/WindowsOnly/BundleInternalTests.mm

```diff
@@ -0,0 +1,88 @@
+﻿//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <TestFramework.h>
+#import <Foundation/Foundation.h>
+#import <objc/encoding.h>
+
+static NSString* _bundleName = @"MyBundle.bundle";
+
+class NSBundleLocalization : public ::testing::Test {
+public:
+    explicit NSBundleLocalization() : ::testing::Test() {
+    }
+
+protected:
+    virtual void SetUp() {
+        NSString* _subDirectory = @"en.lproj";
```

Comment:
?? you're only testing looking strings up in the single lproj that exists, which usually matches the system locale.

Reply:
The point was to always find a localization we have. Since we know we can use English as a fallback the test doesn't care about what locale or language settings the machine has. This way we can ensure we're testing the localization changes.

Reply:
by testing the base localization, we aren't testing anything related to 1. localization or 2. localization changes though.

Reply:
I mean, we're certainly testing the NSLocalizedString path. Grabbing a particular locale is not the goal here, nor is that something I think we can rely on with a unit test. The system can only have one locale setting, so it doesn't seem to matter to me which locale that is. Since English is a known fallback is that not acceptable?